onadata/apps/api/management/commands/delete_users.py

```diff
@@ -0,0 +1,90 @@
+from django.contrib.auth.models import User
+from onadata.apps.logger.models import XForm
+from onadata.apps.logger.models import Instance
+from onadata.apps.logger.models import Project
+from django.core.management.base import BaseCommand
+
+
+class Command(BaseCommand):
```

Comment:
This might be nitpicky, but I favor moving the custom functions within here to somewhere else and then calling them from the handle method.  You can then more easily test them.